src/tests/unit/tests/DetailsView/components/__snapshots__/details-view-command-bar.test.tsx.snap

```diff
@@ -14,197 +14,11 @@ exports[`DetailsViewCommandBar renders with buttons collapsed into a menu 1`] =
       </StyledLinkBase>
     </StyledTooltipHostBase>
   </div>
-  <CommandBarButtonsMenu deps={{...}} tabStoreData={{...}} switcherNavConfiguration={{...}} scanMetadata={{...}} narrowModeStatus={{...}} visualizationStoreData={{...}} unifiedScanResultStoreData={{...}} visualizationConfigurationFactory={[Function: function () {
-                      var args = [];
-                      for (var _i = 0; _i < arguments.length; _i++) {
-                          args[_i] = arguments[_i];
-                      }
-                      _this._interceptor.removeInvocation(invocation_1);
-                      var method = new MethodInfo(target, p);
-                      var methodInvocation = new MethodInvocation(target, method, args, ProxyType.DYNAMIC);
-                      _this._interceptor.intercept(methodInvocation);
-                      return methodInvocation.returnValue;
-                  }]} selectedTest={-1} renderExportReportButton={[Function]} buttonRef={[Function: buttonRef]} />
+  <CommandBarButtonsMenu deps={{...}} tabStoreData={{...}} switcherNavConfiguration={{...}} scanMetadata={{...}} narrowModeStatus={{...}} renderExportReportButton={[Function]} buttonRef={[Function: buttonRef]} />
```

Comment:
This looks to be `debug` snapshot. Lets see if we can use `getElement` instead?

Reply:
Can't, due to a mock being in the snapshot